README.md

```diff
@@ -1,2 +1,33 @@
 # product-landing-page
-5th-project
+5th-project# project\product-landing-page 🚀
+
+## Project Description 📝
+
+
+Examples:
```

Comment:
- Remove just the `Example` text from here as it was just for showing you the Examples.